src/Sentry.Unity.Android/IJniExecutor.cs

```diff
@@ -0,0 +1,10 @@
+using System;
+
+namespace Sentry.Unity.Android
+{
```

Comment:
can u use top level `namespace` here already?

Reply:
I might, but everything in that `.csproj` is inside the `Sentry.Unity.Android` namespace.